doc/content/integrations/application-packages/lora-cloud-device-management.md

```diff
@@ -51,3 +51,39 @@ This will enable the package on FPort `200` of the device `dev1` of application
     }
 }
 ```
+
+## Using a custom server URL
+
+{{< cli-only >}}
+
+The package may be configured to use a custom server URL using the package data:
+
+```bash
+# Create a JSON formatted file containing the uplink token and the server URL
+$ echo '{ "token": "AQEAdqwV67...", "server_url": "https://app.example.com/" }' > package-data.json
```

Comment:
This seems the first occurrence of where a `server_url` key is mentioned.

Don't we need a specification of the package data JSON, to begin with? Noting what the required values are, what optional is and what is default?